docs/components/modeler/web-modeler/advanced-modeling/business-rule-task-linking.md

```diff
@@ -0,0 +1,29 @@
+---
+id: business-rule-task-linking
+title: Business rule task linking
+description: Use one of the following approaches to link the DMN decision to be called by a business rule task.
+---
+
+You can use one of the following approaches to link the DMN decision to be called by a [business rule task](/components/modeler/bpmn/business-rule-tasks/business-rule-tasks.md).
+
+### Using the link button
+
+1. Select a business rule task from the canvas and a link button will appear at the bottom right.
+2. Click on the button and choose any decision from the same project.
+3. Click the **Link** button to complete the linking process. In the properties panel, the value DMN decision is chosen for the **Implementation** property and the Decision ID of the decision you chose to link is automatically copied to the **Called decision** section.
```

Comment:
I'm not sure about this, but I would highlight "DMN decision" because it's an entry of the dropdown.

```suggestion
3. Click the **Link** button to complete the linking process. In the properties panel, the value **DMN decision** is chosen for the **Implementation** property and the Decision ID of the decision you chose to link is automatically copied to the **Called decision** section.
```